servers.json

```diff
@@ -1,4 +1,11 @@
 [
+    {
+        "id": "pvparcade",
```

Comment:
There is already a mapping for a server with the `pvparcade` id, please choose another id.

Reply:
Sorry for the late response, been busy. I completely overlooked this, newest commit should resolve this. 👍